src/phoenix/server/api/mutations/chat_mutations.py

```diff
@@ -79,21 +96,188 @@ class ChatCompletionMutationPayload:
     error_message: Optional[str]
 
 
+@strawberry.type
+class ChatCompletionMutationError:
+    message: str
+
+
+@strawberry.type
+class ChatCompletionOverDatasetMutationExamplePayload:
+    dataset_example_id: GlobalID
+    experiment_run_id: GlobalID
```

Comment:
it doesn't hurt to have this but i don't think we strictly need this here since all examples will be from the same experiment